src/core/loss_detection.c

```diff
@@ -456,6 +463,39 @@ QuicLossDetectionOnPacketSent(
             &Connection->CongestionControl, SentPacket->PacketLength);
     }
 
+    SentPacket->Flags.IsAppLimited = QuicCongestionControlIsAppLimited(&Connection->CongestionControl);
+
+    uint64_t SendPostedBytes = Connection->SendBuffer.PostedBytes;
+
+    CXPLAT_LIST_ENTRY* Entry = Connection->Send.SendStreams.Flink;
+    QUIC_STREAM* Stream = (Entry != &(Connection->Send.SendStreams)) ?
+                          CXPLAT_CONTAINING_RECORD(Entry, QUIC_STREAM, SendLink) :
+                          NULL;
```

Comment:
Generally, our style for these type of multi-line expressions would be like this instead:
```suggestion
    QUIC_STREAM* Stream =
        (Entry != &(Connection->Send.SendStreams)) ?
          CXPLAT_CONTAINING_RECORD(Entry, QUIC_STREAM, SendLink) :
          NULL;
```
Same for all the multi-line expressions you've added. Thanks!